javalib/src/main/scala/java/lang/Runtime.scala

```diff
@@ -81,6 +93,7 @@ class Runtime private () {
       if (isMultithreadingEnabled) runHooksConcurrent()
       else runHooksSequential()
     }
+    exit(0)
```

Comment:
What if the shutdown was started by unhandled thread or signal? In such case we would report successful exit during program failure

Reply:
This one I put in because it was hanging after the shutdown hooks ran and I had no idea how to solve that.